__tests__/articleRoutes.test.js

```diff
@@ -0,0 +1,103 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '..';
+import mockUsers from './mockData/mockUsers';
+import mockArticles from './mockData/mockArticles';
+
+chai.use(chaiHttp);
+
+const API_PREFIX = '/api/v1/articles';
+const { expect } = chai;
+let validUserToken;
+
+describe('Article Routes Test', () => {
+  before((done) => {
+    chai
+      .request(app)
+      .post('/api/v1/auth/signup')
+      .send(mockUsers[9])
+      .end((err, res) => {
+        const { token } = res.body.user;
+        validUserToken = token;
+        done();
+      });
+  });
+
+  it('should throw an error when the title of an article is missing', (done) => {
+    chai.request(app)
+      .post(`${API_PREFIX}`).send(mockArticles[1]).set('Authorization', `Bearer ${validUserToken}`)
+      .end((err, res) => {
+        expect(res.status).to.be.eql(400);
+        expect(res.body).to.have.property('errors');
+        expect(res.body.errors).to.be.to.a('object');
+        expect(res.body.errors).to.have.property('title').eql('Title cannot be empty');
+        done();
+      });
+  });
+
+  it('should throw an error when the tags of an article more than the allowed amount', (done) => {
+    chai.request(app)
+      .post(`${API_PREFIX}`).send(mockArticles[2]).set('Authorization', `Bearer ${validUserToken}`)
+      .end((err, res) => {
+        expect(res.status).to.be.eql(400);
+        expect(res.body).to.have.property('errors');
+        expect(res.body.errors).to.be.to.a('object');
+        expect(res.body.errors).to.have.property('tags').eql('Only a maximum of 3 tags are allowed');
```

Comment:
Only a maximum of 3 tags are allowed?

Reply:
No, this is an outdated commit. It has been increased to 10 now